graylog2-server/src/main/java/org/graylog2/plugin/utilities/date/NaturalDateParser.java

```diff
@@ -58,7 +58,7 @@ public NaturalDateParser(final String timeZone, final Locale locale) throws Ille
 
         this.locale = locale;
         this.timeZone = TimeZone.getTimeZone(timeZone);
-        this.zoneId = ZoneId.of(timeZone);
+        this.zoneId = ZoneId.of(timeZone, ZoneId.SHORT_IDS);
```

Comment:
unfortunately, this alone won't do as the `isValidTimeZone` (line 55) will already reject the short ids

Reply:
Good catch. In that case I think we should not make any change to `NaturalDateParser` at all. Just throw the exception if we encounter a short timezone ID going forward. We should move away from those deprecated IDs anyway. And if we really do need to support them for specific scenarios, we can handle that in the scenario-specific code rather than in a central utility class.